diff --git a/node_modules/@pixi/utils/lib/index.mjs b/node_modules/@pixi/utils/lib/index.mjs
index 74cc6a7..82dfcad 100644
--- a/node_modules/@pixi/utils/lib/index.mjs
+++ b/node_modules/@pixi/utils/lib/index.mjs
@@ -2,7 +2,6 @@ import "./settings.mjs";
 import { isMobile } from "@pixi/settings";
 import { default as default2 } from "eventemitter3";
 import { default as default3 } from "earcut";
-import { url } from "./url.mjs";
 import { path } from "./path.mjs";
 import { detectVideoAlphaMode } from "./browser/detectVideoAlphaMode.mjs";
 import { sayHello, skipHello } from "./browser/hello.mjs";
@@ -68,6 +67,5 @@ export {
   string2hex,
   trimCanvas,
   uid,
-  url
 };
 //# sourceMappingURL=index.mjs.map
